corehq/apps/app_manager/templates/app_manager/partials/modules/case_search_properties.html

```diff
@@ -466,11 +466,23 @@ <h4 class="panel-title panel-title-nolink">{% trans "Search and Claim Options" %
             </div>
           </div>
           {% endif %}
+          {% if request|toggle_enabled:'SPLIT_SCREEN_CASE_SEARCH' and not app.split_screen_dynamic_search%}
```

Comment:
should a check be added to the dynamic search checkbox to check that `app.search_on_clear` is not enabled?

Reply:
I don't think so because dynamic search has behaviors in addition to searching on clear. So app builder may decide to enable dynamic search even after enabling "search on clear" which would be fine.